banyand/measure/merger.go

```diff
@@ -107,7 +108,30 @@ func (tst *tsTable) mergePartsThenSendIntroduction(creator snapshotCreator, part
 	reservedSpace := tst.reserveSpace(parts)
 	defer releaseDiskSpace(reservedSpace)
 	start := time.Now()
+	var totalSize int64
+	for _, pw := range parts {
+		totalSize += int64(pw.p.partMetadata.CompressedSizeBytes)
+	}
+	shouldDropCache := fs.ShouldApplyFadvis(totalSize)
```

Comment:
```suggestion
	
```
Move this piece into "mergeParts":

```
for i := range parts {
		pmi := generatePartMergeIter()
		pmi.mustInitFromPart(parts[i].p)
		pii = append(pii, pmi)
// sum totalSize here
	}
```

Reply:
pass "pm.Protector" to "tsTable" through "measure.option". https://github.com/apache/skywalking-banyandb/blob/main/banyand/measure/measure.go#L46 

```
shouldDropCache := tst.pm.ShouldApplyFadvis(totalSize)
```